### DISCUSSION TITLE
Do I disable treesitter of markdown for telekasten filetype?

### DISCUSSION THREAD

**Discussion by Jaehaks** [nvim-telekasten/telekasten.nvim#338] [Q&A] [answered]
If the filetype is telekasten, markdown treesitter invoke error because it has .md extension but the file type is telekasten.

But the treesitter is depneds by language, not fileytpe 
How do i disable the treesitter for telekasten?

**Comment by lambtho12** [comment, marked as answer]
I think it is best to disable the telekasten filetype instead. It brings more bugs than features :sweat_smile:

**Reply by Jaehaks** [reply]
That's a shame ...
